lib/diagnostic.rb

```diff
@@ -45,7 +56,11 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Developer < Person
+  DEV = Person.new(solve_problems: 'think think think')
```

Comment:
This will cause an error since `Person`'s constructor expects three arguments.

```ruby
class Developer < Person
  def solve_problems
    'think think think'
  end
end
```